Improve Performance counter handling (#1536)
elastic/apm-agent-dotnet: docs/metrics.asciidoc

As of APM version 6.6, these metrics will be visualized in the APM app.

[IMPORTANT]
--

> System metrics are collected using Performance Counters on Windows. The account under which a traced

Reviewer: I think this is not fully accurate and can be a bit misleading. We only collect the system CPU usage with perf. counters, e.g. system memory (`system.memory.actual.free` and `system.memory.total`) use a win32 call.

```suggestion
System CPU usage is collected using Performance Counters on Windows. The account under which a traced
```
